core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/util/UtilV4Endpoint.java

```diff
@@ -49,7 +49,10 @@
     @Path("stack_matrix")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = UtilityOpDescription.STACK_MATRIX, produces = MediaType.APPLICATION_JSON, nickname = "getStackMatrixUtilV4")
-    StackMatrixV4Response getStackMatrix(@QueryParam("imageCatalogName") String imageCatalogName, @QueryParam("platform") String platform) throws Exception;
+    StackMatrixV4Response getStackMatrix(
+        @QueryParam("imageCatalogName") String imageCatalogName,
+        @QueryParam("platform") String platform,
+        @QueryParam("govCloud") boolean govCloud) throws Exception;
```

Comment:
Shouldn't it be more maintainable in the future if we introduce a variant field instead of the govCloud boolean?

Reply:
we just discussed